src/Databricks/Az.Databricks.psd1

```diff
@@ -1,133 +1,24 @@
-#
-# Module manifest for module 'Az.Databricks'
-#
-# Generated by: Microsoft Corporation
-#
-# Generated on: 3/24/2020
-#
-
 @{
```

Comment:
this file needs to be changed manually. Given no new cmdlet, it needs to be rollback